src/pages/guides/theming/index.md

```diff
@@ -0,0 +1,434 @@
+- **theme** - Defines your custom values that override or extend the Tailwind CSS framework. See [Theme Configuration](https://v2.tailwindcss.com/docs/theme) from the Tailwind documentation.
+- **matcher** - Matches CSS Modules classnames that use the `composes` function. PurgeCSS uses this value to remove unused CSS from the `index.css` file. See [Writing purgeable HTML](https://v2.tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html) from the Tailwind documentation.
+
+<InlineAlert slots="text" />
+
+When looking at code samples in the Tailwind docs, keep in mind that the base project configures Tailwind to use `_` as the [separator](https://v2.tailwindcss.com/docs/configuration#separator) instead of the default colon (`:`).
+
+### postcss.config.js
+
+Create the `postcss.config.js` file in your project's root and configure it to add `tailwindcss` and the `autoprefixer` as PostCSS plugins for your app. Your `postcss.config.js` file should look like this:
+
+```js
+// postcss.config.js
+
+module.exports = {
+  plugins: [
+    require('autoprefixer'),
+    require('tailwindcss')('./tailwind.config.js'),
+  ],
+};
+```
+
+## Test theme configuration
+
+For a quick test of your Venia configuration, add the following `theme` override directly to `tailwind.config.js`, then remove it after you verify it works. If the text color in your Venia-themed app turns red, your configuration is working correctly.
+
+```js:
+
+```js
+// tailwind.config.js
+
+theme: {
+  extend: {
+    textColor: {
+      colorDefault: '#BB0000'
+    }
+  }
+}
+```
+
+To test your own theme, refer to [Customizing your theme](https://tailwindcss.com/docs/adding-custom-styles#customizing-your-theme) from the Tailwind documentation.
+
+## Customize base styles
+
+You have two options for customizing the Tailwind and Venia base styles:
+
+1. Update the `tailwind.config.js` file directly.
+1. Import your own theme file as a `preset` (recommended).
+
+### Update tailwind.config.js directly
+
+As you did to test the theme configuration, you can override or extend the Tailwind and Venia themes directly within the `theme` property of the `tailwind.config.js.` file as shown here:
+
+```js
+// tailwind.config.js
+
+theme: {
+  extend: {
+    fontFamily: {
+      sans: ['"Open Sans"', 'sans-serif']
+    }
+    backgroundColor: {
+      subtitle: '#F5F5F5',
+    }
+    borderRadius: {
+      radius4: '50%',
+    }
+    ...
+  }
+}
+```
+
+However, you should avoid this option for all but the smallest real-world changes. For example, if you plan to use the Venia base theme with only a few custom colors and fonts, this option may be fine for your needs. However, if you are going to make several style changes or plan to rotate through different themes (seasonal or otherwise), use option 2.
```

Comment:
```suggestion
However, you should avoid this option for all but the smallest real-world changes. For example, if you plan to use the Venia base theme with only a few custom colors and fonts, this option may be fine for your needs. However, if you are going to make several style changes or plan to rotate through different themes (seasonal or otherwise), create a new theme file instead.
```